ClemBot.Bot/bot/services/emote_board_service.py

```diff
@@ -101,10 +101,15 @@ async def on_message_edit(self, event: RawMessageUpdateEvent) -> None:
         guild = self.bot.get_guild(event.guild_id)
         assert guild is not None
 
-        channel = guild.get_channel(event.channel_id)
+        channel = guild.get_channel_or_thread(event.channel_id)
         assert channel is not None and isinstance(channel, discord.abc.Messageable)
 
-        message = await channel.fetch_message(event.message_id)
+        # Attempt to fetch the message, ignore if we do not have permissions.
+        message: discord.Message
+        try:
+            message = await channel.fetch_message(event.message_id)
```

Comment:
we can still link it right? just have the comment say "bot not permitted to read message" or something

Reply:
Not that I'm aware of. It's honestly kind of a weird edge case... We can see the message was edited, but we cannot read the message itself... If we can't read the message, we can't read the reactions, and we cannot update the embed if we cannot read the message content. Honestly not sure how this came about.

Reply:
might not have granted the bot message content in oauth perms

Reply:
probably need a log statement here so we know it happens

info level log